pom.xml

```diff
@@ -545,16 +557,14 @@
     </profile>
     <profile>
       <id>spark-2.2</id>
-      <activation>
```

Comment:
It seems don't support "mvn clean package" command if you remove this.

Reply:
Because activeByDefault setting limit maven profile only spark-2.2.If there is activeByDefault setting,Intellij idea perform switch profile only add active profile.The default active profile dependencies always exists.But this is not necessary that removing activeByDefault setting is helpful for switching profile in Intellij idea.